x-pack/test/functional_cloud/config.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { resolve } from 'path';
+import { FtrConfigProviderContext } from '@kbn/test';
+
+export default async function ({ readConfigFile }: FtrConfigProviderContext) {
+  const functionalConfig = await readConfigFile(require.resolve('../functional/config.base.js'));
+
+  const kibanaPort = functionalConfig.get('servers.kibana.port');
+  // const idpPath = resolve(__dirname, '../security_api_integration/fixtures/saml/idp_metadata.xml');
+  const idpPath = resolve(
+    __dirname,
+    '../cloud_integration/fixtures/saml/saml_provider/metadata.xml'
+  );
+  const samlIdPPlugin = resolve(__dirname, '../cloud_integration/fixtures/saml/saml_provider');
+
+  return {
+    ...functionalConfig.getAll(),
+    rootTags: ['skipCloud'],
+    testFiles: [require.resolve('./tests')],
+    security: { disableTestUser: true },
+    junit: {
+      reportName: 'Cloud Integrations Functional Tests',
+    },
+    esTestCluster: {
+      ...functionalConfig.get('esTestCluster'),
+      serverArgs: [
+        ...functionalConfig.get('esTestCluster.serverArgs'),
+        'xpack.security.authc.token.enabled=true',
+        'xpack.security.authc.token.timeout=15s',
```

Comment:
__nit:__ it's not actually required for your tests (it's used to test session timeouts)
```suggestion
```